tests/unit/components/LemmaInput.test.ts

```diff
@@ -14,7 +14,7 @@ describe( 'LemmaInput', () => {
 
 	let store: Store<RootState>;
 
-	function createComponent( config: Record<string, unknown> = {} ) {
+	function createComponent( config: Record<string, unknown> = { maxLemmaLength: 8 } ) {
```

Comment:
setting it here doesn't seem to make a difference, but that could also mean the validation isn't actually working